test/parallel/test-http-outgoing-writableFinished.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
This is already merged in commit https://github.com/nodejs/node/commit/7032e59bb550adcb448b2d24877921e7f97e7399 and shouldn't be part of this PR